project-management/src/main/java/life/qbic/projectmanagement/application/api/AsyncProjectService.java

```diff
@@ -768,6 +770,33 @@ record ProjectCreationRequest(ProjectDesign design, ProjectContacts contacts,
 
   }
 
+  /**
+   * Represents an ontology term definition with a simple label that can be used to display the term
+   * for humans, its assigned OBO identifier and its globally unique identifier.
+   *
+   * @param label an {@link OntologyTerm} label for visualisation
+   * @param oboId the assigned OBO identifier
+   * @param id    the globally unique identifier of the term
+   * @since 1.10.0
+   */
+  record OntologyTerm(String label, Curie oboId, URI id) {
+
+  }
```

Comment:
In this case it could be useful to introduce null checks at least for the `Curie`